net/interface.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type OrchestratorSelector interface {
-	GetOrchestrators(int) ([]*OrchestratorInfo, error)
+	GetOrchestrators(int) ([][]*OrchestratorInfo, error)
```

Comment:
The interface should stay the same. The other changes to `offchainOrchestrators` is what's enabling us to return more than one `OrchestratorInfo` here.